src/core/trulens/core/utils/trulens.py

```diff
@@ -34,49 +31,3 @@ def component_of_json(json: dict) -> base.TrulensComponent:
     view = constructor_of_class(cls)
 
     return view(json)
```

Comment:
This is part of packaging change?

Reply:
@coreyhu can you verify?

Reply:
sorry not sure what's being referred to here

Reply:
@sfc-gh-pdharmana there's diff in the code you're commenting on. Can you clarify?